libvast/vast/view.hpp

```diff
@@ -169,6 +174,18 @@ struct view_trait<data> {
   using type = data_view;
 };
 
+// -- operators ----------------------------------------------------------------
+
+// FIXME: we cannot use operator == and != here because data has a non-explicit
+// constructor, which results in error all over the code base. Therefore, we
+// work around this by giving this function a name.
```

Comment:
Do _you_ intend to fix this? Otherwise this should be a `TODO` and an associated story in ClubHouse so we don't forget about it.

Reply:
Demoted to `TODO`.